deps/build.jl

```diff
@@ -70,4 +70,8 @@ provides(BuildProcess, (@build_steps begin
     end
 end), [libfftw, libfftwf])
 
-BinDeps.@install Dict([:libfftw => :libfftw, :libfftwf => :libfftwf])
+if is_windows()
+    BinDeps.@install Dict([:libfftw => :libfftw3, :libfftwf => :libfftwf3])
+else
+    BinDeps.@install Dict([:libfftw => :libfftw3_threads, :libfftwf => :libfftwf3_threads])
```

Comment:
I think these are supposed to be the variable name for the library_dependency within this file and the variable name that it should have inside deps.jl, not the value of what that name should be